.pre-commit-config.yaml

```diff
@@ -37,6 +33,14 @@
         - --add-import
         - from __future__ import unicode_literals
 -   repo: https://github.com/asottile/pyupgrade
-    sha: v1.1.1
+    rev: v1.22.1
     hooks:
     -   id: pyupgrade
+-   repo: git@git.yelpcorp.com:mirrors/pre-commit/mirrors-autopep8
```

Comment:
As you've probably seen yourself, this needs to be a public URL.

Reply:
copy-paste fail :(